src/phasorpy/phasor.py

```diff
@@ -1031,6 +1033,7 @@ def phasor_calibrate(
     ValueError
         The array shapes of `real` and `imag`, or `reference_real` and
         `reference_imag` do not match.
+        Frequency shape does not match the shape of `real` along `skip_axis`.
```

Comment:
Frequency/harmonics is not the only axis that can be skipped. In fact, one might want to skip anything but spatial dimensions, for example, channels and time-axis in addition to frequency.

Reply:
The verification of combinations of `skip_axis`, `frequency`, `harmonic`, and `real`/`image` parameters is going to be somewhat tedious. It's probably worth to first write down on paper which cases are supported, then write tests, and then adjust the implementation.